src/status_im/components/list_selection.cljs

```diff
@@ -37,9 +37,10 @@
                                        (case index
                                          0 (do
                                              (dispatch [:select-chat-input-command
-                                                        {:bot     "browse"
-                                                         :prefill [link]}])
-                                             (js/setTimeout #(dispatch [:send-current-message]) 500))
+                                                        (assoc (first command) :prefill [link])
+                                                        nil
+                                                        true])
+                                             (js/setTimeout #(dispatch [:send-current-message]) 100))
```

Comment:
Ideally we should get rid of such horrible delayed dispatch code, given then we are short on time before release, we can go with it, but please add `TODO` comment to refactor it in the future.

Reply:
I completely agree